CHANGELOG/CHANGELOG-2.3.md

```diff
@@ -0,0 +1,58 @@
+- Support version skew between Antrea Agent and Flow Aggregator to improve upgrade robustness. ([#6912](https://github.com/antrea-io/antrea/pull/6912), [@antoninbas])
+- Add `clusterId` to aggregated records for Flow Aggregator. ([#6769](https://github.com/antrea-io/antrea/pull/6769), [@antoninbas])
+- Add `checksum/config` annotation to the Deployment of Flow Aggregator. ([#6967](https://github.com/antrea-io/antrea/pull/6967), [@antoninbas])
+- Add more fields in `BGPPolicy` API to support BGP confederation. ([#6905](https://github.com/antrea-io/antrea/pull/6905), [@hongliangl])
+- Support SecondaryNetwork of `SR-IOV` type for VM Nodes. ([#6881](https://github.com/antrea-io/antrea/pull/6881), [@tnqn])
+- Add more printer columns for `PacketCapture` CRD. ([#6977](https://github.com/antrea-io/antrea/pull/6977), [@antoninbas])
+- Add logs collection for failed agents and controller for Support Bundle. ([#3659](https://github.com/antrea-io/antrea/pull/3659), [@hangyan])
+- Add `antreaProxy.disableServiceHealthCheckServer` config to disable the health check server run by Antrea Proxy to avoid `kube-proxy` error logs. ([#6939](https://github.com/antrea-io/antrea/pull/6939), [@antoninbas])
+- Add route info in the output of `antctl get bgproutes`. ([#6803](https://github.com/antrea-io/antrea/pull/6803) [#6835](https://github.com/antrea-io/antrea/pull/6835), [@Atish-iaf])
+
+### Changed
+
+- Promote feature `EgressSeparateSubnet` from Alpha to Beta. ([#6982](https://github.com/antrea-io/antrea/pull/6982), [@luolanzone])
+- Promote feature `ServiceExternalIP` from Alpha to Beta. ([#6903](https://github.com/antrea-io/antrea/pull/6903), [@xliuxu])
+- Accept configurations with no collector / sink to avoid crash issue for Flow Aggregator. ([#7006](https://github.com/antrea-io/antrea/pull/7006), [@antoninbas])
+- More efficient IP check to query `NodeRouteController` for storing all the PodCIDRs to determine whether the source / destination IPs are Pod IPs (NodeIPAM only) for `FlowExporter`. ([#6960](https://github.com/antrea-io/antrea/pull/6960), [@antoninbas])
+- Remove stale OVS interfaces in the CNIServer reconciler if the original Pod interface is disconnected. ([#6919](https://github.com/antrea-io/antrea/pull/6919), [@wenyingd])
+- Remove local ASN range limitation in `BGPPolicy` API. ([#6914](https://github.com/antrea-io/antrea/pull/6914), [@hongliangl])
+- Support providing a fixed public host key for SFTP uploads with a new field `hostPublicKey` to `PacketCapture` and `SupportBundleCollection` CRDs. ([#6848](https://github.com/antrea-io/antrea/pull/6848), [@antoninbas])
+- Skip processing Service without IPPool annotation `service.antrea.io/external-ip-pool` for `ServiceExternalIP` feature. ([#6950](https://github.com/antrea-io/antrea/pull/6950), [@xliuxu])
```

Comment:
This doesn't have real impact to users, maybe no need to mention.

Reply:
Removed